docs/source/llms/chat-model-intro/chat-model-vs-pyfunc-table.rst

```diff
@@ -0,0 +1,19 @@
+.. list-table::
```

Comment:
just writing this down so i remember for the docusaurus port:

https://docusaurus.io/docs/markdown-features/react#importing-markdown

looks like we can import markdown from other files, so it should be straightforward to migrate as-is. however, since we also use this table in the API docstrings, we will eventually have two sources of truth

Reply:
Great point. I think having two are not too bad, but we should think about a solution when we need to add it to more docstrings.